modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,51 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
+
+For example, you can have an API specification that contains multiple examples within a response as shown in the following example:
+
+----
+/examples:
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            examples:
+              one:
+                prop1: something
+                prop2: 30
+              two:
+                prop1: another
+                prop2: 31
+              three:
+                prop1: anything
+                prop2: 33
+----
+
+Here, the possible values of the header would be `one`, `two`, and `three`. Sending the value `one` as part of the request would return the payload `something` and `30`. Sending a value other than `one`, `two`, and `three` would return an error message.
```

Comment:
"Sending a value other than `one`, `two`, and `three`" -> "Sending a value other than `one`, `two`, or `three`"

Reply:
Thanks! Fixed!